.vscode/settings.json

```diff
@@ -26,6 +26,5 @@
     "typings",
     "**/*.snap",
     "**/*.story.tsx"
-  ],
-  "editor.formatOnSave": true
```

Comment:
@mzikherman - this removes all of the prettier / tslint autofix updates on save; why remove?